code/modules/events/immovablerod.dm

```diff
@@ -101,9 +103,23 @@
 		qdel(src)
 		return
 
-	if(loc.density)
-		loc.ex_act(2)
-		if(prob(25))
+	if(loc && !istype(loc,/turf/space))
+		if(fuckMyShitUp)
+			loc.ex_act(1)
+		else if(loc.density)
+			if(istype(loc,/turf/simulated/wall/r_wall))
+				loc.ex_act(1)
```

Comment:
@ComicIronic r_walls can tank ex_act(1) pretty well. Half the time the wall is only partially destroyed even. It never creates a hole into space. Unlike what doing a Destroy (or qdel rather) would do.